routes/api.php

```diff
@@ -157,5 +157,6 @@
     Route::post('support/messages/send', 'Support\Messages\SendingController');
 });
 
+Route::match(['get', 'post'], 'payment_webhook/{company_key}/{gateway_key}', 'PaymentWebhookController');
```

Comment:
We need to make sure that we set the DB correctly when using multidb, i think a middleware which resolves the company_key should solve this.